src/AnomalyDetectors/OneClassSVM.php

```diff
@@ -182,7 +183,13 @@ public function train(Dataset $dataset) : void
             new SamplesAreCompatibleWithEstimator($dataset, $this),
         ])->check();
 
-        $this->model = $this->svm->train($dataset->samples());
+        $data = [];
+
+        foreach ($dataset->samples() as $i => $sample) {
```

Comment:
Looks like we don't need this `$i` variable

Reply:
You are completely right, this is now corrected !